src/data.js

```diff
@@ -0,0 +1,63 @@
+const populateList = (obj, ul) => {
+  const users = obj.result;
+  users.forEach((user) => {
+    ul.insertAdjacentHTML('beforeend',
+      `<li class="leaderboard-item" data-score=${user.score}>`
+      + `${user.user} : ${user.score}`
+    + '</li>');
+  });
+};
+
+export const loadList = async (gameId, ul) => {
+  const requestURL = `https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/${gameId}/scores/`;
+  const request = new Request(requestURL);
+  const response = await fetch(request);
+  const users = await response.json();
+  populateList(users, ul);
+};
+
+const addData = async (gameId, obj) => {
+  const url = `https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/${gameId}/scores/`;
+  const options = {
+    method: 'POST',
```

Comment:
- [Optional ] Please make sure to provide a validating for name and score length 
![image](https://user-images.githubusercontent.com/74744787/209102701-7792b991-f8fd-4c03-b6c3-627aa43cc56a.png)